streaming/text/c4.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Dict, Optional
+
+from transformers.models.auto.tokenization_auto import AutoTokenizer
+
+from ..base import Dataset
```

Comment:
Not a blocker but we should decide on `relative` vs `absolute` import consistency. PEP8 recommends absolute imports and the same we have in `composer`.

Reply:
absolute imports it is